hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/AbstractCodeCache.java

```diff
@@ -64,6 +64,11 @@ public Code getIfPresent(final Address address) {
             cache.put(cacheKey, code);
         }
 
+        if (code == null) {
+            code = CodeFactory.createCode(Bytes.EMPTY, 0, false);
+            cache.put(cacheKey, code);
+        }
+
```

Comment:
We should make sure that actually creating a contract on the address would update the code cache with the contract's code and add such an EET.